tasks/Game of Clue - Terrorist Attack.md

```diff
@@ -0,0 +1,38 @@
+# Summary
+Participants played a 'who-done-it' game like Clue or Cluedo in wich the task involved piecing together clues, or facts to connect the dots. Specifically, participants were faced with four logically independent subproblems to solve: (a) who would carry out the attack (group involved), (b) what would be the target (e.g., an embassy or a church), (c) where the attack would take place (country), and (d) when the attack would take place (with four interdependent components—month, day, hour, and time of day (a.m. versus p.m.)). 
+
+# References
+Main Paper: https://pubsonline.informs.org/doi/full/10.1287/orsc.2015.0980#_i34
+
+Supplemental Materials: https://pubsonline.informs.org/doi/suppl/10.1287/orsc.2015.0980/suppl_file/orsc.2015.0980-sm-online_appendix_.pdf
+
+# Stimuli
+## The visual components
+See supplemetal materials PDF for images of computer interface 
+
+
```

Comment:
For materials for other alternate versions:
- There were four 16-person network treatments
- 1) Caveman (CAVE): contained four four-person cliques
- 2) Hierarchy (HIER): four four-person cliques arranged in a conventional centralized structure
- 3) Rewired Caveman (RCAVE): removed links from the caveman network, then added links that created shortcuts through the network
- 4) Ring (RING): a network that was neither clustered nor centralized

Reply:
I thought these were more of the treatment groups within the experiment, then they compared results across the different network structures. I can add in to the procedure section but I'm not sure they would be considered separate experiments?

Reply:
sounds good!